### PR TITLE
Add fops for dfs_v2 and fix warning for eventfd_write and eventfd_read after dfs_v1 changes

### PR DESCRIPTION
**Pull request by zmq810150896**
#### 为什么提交这份PR (why to submit this PR)
添加dfs_v2的fops和解决dfs_v1更改之后eventfd_write和eventfd_read报warning的问题
#### 在什么测试环境下测试通过 (what is the test environment)
qemu-virt64-aarch64
### 当前拉取/合并请求的状态 Intent for your PR

必须选择一项 Choose one (Mandatory):

- [ ] 本拉取/合并请求是一个草稿版本 This PR is for a code-review and is intended to get feedback
- [x] 本拉取/合并请求是一个成熟版本 This PR is mature, and ready to be integrated into the repo

### 代码质量 Code Quality：

我在这个拉取/合并请求中已经考虑了 As part of this pull request, I've considered the following:

- [x] 已经仔细查看过代码改动的对比 Already check the difference between PR and old code
- [x] 代码风格正确，包括缩进空格，命名及其他风格 Style guide is adhered to, including spacing, naming and other styles
- [x] 没有垃圾代码，代码尽量精简，不包含`#if 0`代码，不包含已经被注释了的代码 All redundant code is removed and cleaned up
- [x] 所有变更均有原因及合理的，并且不会影响到其他软件组件代码或BSP All modifications are justified and not affect other components or BSP
- [x] 对难懂代码均提供对应的注释 I've commented appropriately where code is tricky
- [x] 代码是高质量的 Code in this PR is of high quality
- [x] 已经使用[formatting](https://github.com/mysterywolf/formatting) 等源码格式化工具确保格式符合[RT-Thread代码规范](https://github.com/RT-Thread/rt-thread/blob/master/documentation/contribution_guide/coding_style_cn.md) This PR complies with [RT-Thread code specification](https://github.com/RT-Thread/rt-thread/blob/master/documentation/contribution_guide/coding_style_en.md)

### CODE REVIEW

File: components/libc/posix/io/eventfd/eventfd.c [hunk 1]
```diff
@@ -247,6 +249,9 @@ static int do_eventfd(unsigned int count, int flags)
     if (flags & ~EFD_FLAGS_SET)
         return -RT_EINVAL;
 
+    if (count < 0)
+        return -RT_EINVAL;
```
Comment (mysterywolf) [inline components/libc/posix/io/eventfd/eventfd.c:253]: 这块应该返回-EINVAL

**Review by mysterywolf — CHANGES_REQUESTED** [review]

File: components/libc/posix/io/eventfd/eventfd.c (hunk 1 above)
Comment (zmq810150896) [reply components/libc/posix/io/eventfd/eventfd.c]: 加错了，已经改了

File: components/libc/posix/io/eventfd/eventfd.c [hunk 2]
```diff
@@ -257,12 +262,14 @@ static int do_eventfd(unsigned int count, int flags)
         if (status < 0)
         {
             fd_release(fd);
-            ret = status;
+            rt_set_errno(status);
```
Comment (mysterywolf) [inline components/libc/posix/io/eventfd/eventfd.c:265]: rt_set_errno设置的是错误码正值，返回值是赋值，涉及到内核用RT_Exx, POSIX相关是Exxx

**Review by mysterywolf — CHANGES_REQUESTED** [review]

File: components/libc/posix/io/eventfd/eventfd.c (hunk 2 above)
Comment (zmq810150896) [reply components/libc/posix/io/eventfd/eventfd.c:265]: 好，我改一下，我看人家的代码rt_set_errno设置的都是负值

**Review by BernardXiong — APPROVED** [review]

**Pull request closed without merging**